src/common/src/system_param/mod.rs

```diff
@@ -387,23 +389,50 @@ macro_rules! impl_system_params_for_test {
             ret.backup_storage_url = Some("memory-isolated-for-test".into());
             ret.backup_storage_directory = Some("backup".into());
             ret.use_new_object_prefix_strategy = Some(false);
-            ret.time_travel_retention_ms = Some(0);
```

Comment:
I understand that removing this line is necessary to pass the config validation. But seems like 0 is a legal value, which might be used for testing purpose? https://github.com/risingwavelabs/risingwave/blob/e1ae4d45f1e719277af80ce489003c9c49f74367/src/meta/src/hummock/manager/time_travel.rs#L528-L536 should we change the validation logic for `time_travel_retention_ms`? 
https://github.com/risingwavelabs/risingwave/blob/e1ae4d45f1e719277af80ce489003c9c49f74367/src/common/src/system_param/mod.rs#L429-L438
Need confirm from @zwang28 .

Reply:
0 is used to disable time travel.

Please help modfiy the validation rule to permit 0.

please don't remove the line in the test.